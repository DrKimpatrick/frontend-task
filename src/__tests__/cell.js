import React from 'react';
import { shallow } from 'enzyme';
import Cell from '../components/cell';

describe('Cell component', () => {
  it('Should render cell component', () => {
    const component = shallow(<Cell />);
  
    expect(component).toMatchSnapshot();
  });
  
});