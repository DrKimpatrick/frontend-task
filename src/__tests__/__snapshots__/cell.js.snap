// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cell component Should render cell component 1`] = `
<img
  className="square"
  onClick={[Function]}
  src="white.png"
  style={
    Object {
      "width": 60,
    }
  }
/>
`;
